docs/animations/ViewportBehavior.md

```diff
@@ -0,0 +1,60 @@
+# ViewportBehavior
```

Comment:
Please make sure to use the [template](https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/blob/master/docs/.template.md) and add the missing sections, specifically the header

Reply:
Done.